packages/labs/ssr/src/lib/module-loader.ts

```diff
@@ -0,0 +1,279 @@
+/**
```

Comment:
Would be nice in future to do a rename commit separately. For this PR, GitHub doesn't understand that there was a move, so the diff is useless.

Reply:
Ah shoot, sorry